GOVERNANCE.md

```diff
@@ -0,0 +1,55 @@
+# Governance
+
+This document describes the governance process under which the Serverless
+Working Group (WG) will manage this repository.
+
+## Working Group Meetings
+
+In order to provide equitable rights to all Working Group members,
+the following process will be followed:
+
+* Official WG meetings will be announced at least a week in advance.
+* Proposed changes to any document will be done via a Pull Request (PR).
+* PRs will be reviewed during official WG meetings.
+* During meetings, priority will be given to PRs that appear to be ready for
+  a vote over those that appear to require discussions.
+* PRs should not be merged if they have had substantial changes made within
+  two days of the meeting.
+  Rebases, typo fixes, etc. do not count as substantial.
+  Note, administrivia PRs that do not materially modify WG output documents
+  may be processed by WG admins as needed.
+* Resolving PRs ("merging" or "closing with no action") will be done as a
+  result of a motion made during a WG meeting, and approved.
+* Reopening a PR can be done if new information is made available, and a
+  motion to do so is approved.
+* Any motion that does not have "unanimous consent" will result in a formal
+  vote. See [Voting](#voting).
+
+## PRs
+
+Typically, PRs are expected to meet the following criteria prior to being
+merged:
+
+* The author of the PR indicates that it is ready for review by asking for it
+  to be discussed in an upcoming meeting - eg. by adding it to the agenda
+  document.
+* All comments have been addressed.
+* PRs that have objections/concerns will be discussed off-line by interested
+  parties. A resolution, updated PR, will be expected from those talks.
+
+## Voting
```

Comment:
Here is a proposed rewrite of the Voting section to take into account when  primary members may not be able to attend.

## Voting

If a vote is taken during a WG meeting, the follow rules will be followed:

* Only WG members with voting rights will be allowed to vote.
* Only one WG member per company is designated primary and can vote.
* A vote passes if more than 50% of the votes cast approve the motion.
* Only "yes" or "no" votes count, "abstain" votes do not count towards the
  total.
* WG members attain voting rights by attending 3 out of the last 4 WG
  meetings.  They obtain rights after the 3rd meeting, not during.
* A primary may designate a secondary company attendee to attend and still
  be counted for attendance voting rights. This allows for times when the
  primary may not be able to attend.
* For purposes of a vote, a secondary can vote when the primary is not
  in attendance.
* Meeting attendence will be formally tracked [here](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit#gid=0).
  Members must acknowledge their presence verbally, meaning, adding yourself
  to the "Attendees" section of the Agenda document is not sufficient.
  Secondary members should indicate their role at this time.

Reply:
i've updated the PR with the above + some slight tweaks based on a chat Mark and I had.  So removing this comment to avoid confusion as to the latest version of the proposal.